lua/core/init.lua

```diff
@@ -1,6 +1,9 @@
 -- add binaries installed by mason.nvim to path
 local is_windows = vim.loop.os_uname().sysname == "Windows_NT"
-vim.env.PATH = vim.env.PATH .. (is_windows and ";" or ":") .. vim.fn.stdpath "data" .. "/mason/bin"
+vim.env.PATH = vim.env.PATH
+  .. (is_windows and ";" or ":")
+  .. vim.fn.stdpath(vim.g.vim_version > 7 and "state" or "data")
+  .. "/mason/bin"
```

Comment:
This is to add the path of `mason.nvim` to the PATH env, and by default, it is always added to `vim.fn.stdpath("data")`, so no need to check for it here

Reply:
Not to mention doing this renders 0.8.0 users complete broken installs for packages downloaded by mason, as you have to use absolute paths to get the packages, re-add the binary path to `$PATH`, or change default install directory of mason to the path you set here